skopt/optimizer/base.py

```diff
@@ -82,60 +60,39 @@ def gp_minimize(func, dimensions, base_estimator=None, alpha=10e-10,
           `Categorical`).
 
     * `base_estimator` [a Gaussian process estimator]:
-        The Gaussian process estimator to use for optimization.
-        By default, a Matern kernel is used with the following
-        hyperparameters tuned.
-        - All the length scales of the Matern kernel.
-        - The covariance amplitude that each element is multiplied with.
-        - Noise that is added to the matern kernel. The noise is assumed
-          to be iid gaussian.
+        Should be a sklearn Regressor. In addition, should have an optional
```

Comment:
I would say that it should implement the scikit-learn `RegressorMixin` interface.